### DISCUSSION TITLE
force an app open in floating mode

### DISCUSSION THREAD

**Discussion by smjt2000** [qtile/qtile#4038] [Q&A]
Hi
I'm not good in English
I need to open some apps in floating mode(like some KDE apps).
I use KDEconnect app, when I'm in a no floating layout like Columns and want to use presentation cursor in KDEconnect, all opened apps will go to smaller size and cursor open in column layout.

sometimes I want to show something on my screen to students in class, and it's not good.

if I need to know app name to solve the problem help to do this, I don't know how to find app name !!

before using cursor:
![Screenshot_20221205_014050](https://user-images.githubusercontent.com/60272086/205519658-fdb02ec7-02af-417f-920a-39d2a9ca2101.png)

while using cursor:
![Screenshot_20221205_014100](https://user-images.githubusercontent.com/60272086/205519701-bf175aca-bf22-4e5a-ae94-fe23bc1f1faf.png)

**Comment by jwijenbergh** [comment]
You need to change this part of your config https://github.com/qtile/qtile/blob/3225a562e0b0d573294dc140631399076a018d1f/libqtile/resources/default_config.py#L168

Run xprop and click on the window, then get the class name/title, you would then add it to the list like the rest. This makes the window automatically floating

**Comment by Hammer2900** [comment]
Key([], 'F7', lazy.spawn('sakura --class FloatTerminal')),

Match(wm_class='FloatTerminal'),
